tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -0,0 +1,191 @@
+import { VALIDATION_ERROR } from 'const';
+import { CONTENT_SCRIPT_NOT_FOUND, CONTENT_SCRIPT_EMPTY } from 'messages/javascript';
+import JavaScriptScanner from 'scanners/javascript';
+
+describe('content_scripts_file_absent', () => {
+  it('should show an error when content script is missing', () => {
+    const code = `browser.tabs.executeScript({file: '/content_scripts/absentFile.js'});`;
```

Comment:
Nit, it seems that all the tests in this test file are sharing some of the "test case setup steps"  (e.g. like the code between line 7 and line 16 of this test case), it could be reasonable to refactor it into a small shared helper function defined and used in this test file, something like:

```js
function createJsScanner(code, validatedFilename, existingFiles = {}) {
  return new JavascriptScanner(code, validatedFileName, {
    addonMetadata: { id: 'test' },
    existingFiles: {
       ...existingFiles,
       [validatedFilename]: ''
    },
  });
}
```

Then using the helper every test case could be reduce to something like:

```js
it('should ...', async () => {
  const code = `...`
  const jsScanner = createJsScanner(code, 'validated-file-name.js');
  // or createJsScanner(code, 'validated-file-name.js', {'another/file/name.js': ''});
  const { linterMessages } = await jsScanner.run();
  ... // assertions on the linterMessages
});
```